KUBEFLOW-STEERING-COMMITTEE.md

```diff
@@ -43,59 +43,14 @@ Seats on the Steering Committee are held by an individual, not by their employer
 
 The current membership of the committee is (listed alphabetically by first name):
 
-<table>
-  <thead>
-    <tr>
-      <th><br>
-Member</th>
-      <th><br>
-Organization</th>
-      <th><br>
-Github Profile</th>
-      <th><br>
-Term Start</th>
-      <th><br>
-Term End</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-      <td></td>
-    </tr>
-  </tbody>
-</table>
+| Name                | Organization | GitHub                                             | Term Start | Term End   |
+|---------------------|--------------|----------------------------------------------------|------------|------------|
+| Andrey Velichkevich | Apple        | [andreyvelich](https://github.com/andreyvelich/)   | 02/01/2024 | 02/01/2026 |
+| Johnu George        | Nutanix      | [johnugeorge](https://github.com/johnugeorge/)     | 02/01/2024 | 02/01/2026 |
+| Josh Bottum         | Independent  | [jbottum](https://github.com/jbottum/)             | 02/01/2024 | 02/01/2025 |
+| James Wu            | Google       | [james-jwu](https://github.com/james-jwu/)         | 02/01/2024 | 02/01/2025 |
+| Yuan Tang           | Red Hat      | [terrytangyuan](https://github.com/terrytangyuan/) | 02/01/2024 | 02/01/2026 |
```

Comment:
@kubeflow/kubeflow-steering-committee Please check.

@akgraner What should Josh and James' start term be?

Reply:
Josh and I should have the same start term as other KSC members. End term looks correct.

/lgtm